Bottom: 53337fdfe43cd0acc001714fd5ea374a5ffe000a
Top:    4c9d78db8f06f127eae8fadea101d6dbb5e6741f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-26 16:28:35 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
